game/ProfessionPage/ChooseProfessionStep.gd

```diff
@@ -0,0 +1,96 @@
+extends Control
+ 
+
+signal profession_chosen(current_profession)
+signal trait_chosen(profession_trait_element)
+signal clear_trait()
+
+
+export(ButtonGroup) var trait_group
+
+
+var profession_list = []
+var current_profession = 0
+var profession = {}
+
+
+export (NodePath) onready var picture = get_node(picture) as TextureRect
+export (NodePath) onready var profession_name = get_node(profession_name) as RichTextLabel
+export (NodePath) onready var profession_description = get_node(profession_description) as RichTextLabel
+export (NodePath) onready var trait_container =  get_node(trait_container) as HBoxContainer
+export (NodePath) onready var profession_quote = get_node(profession_quote) as RichTextLabel
+
+onready var trait_button_scene = preload("res://ProfessionPage/ProfessionTrait.tscn")
+onready var db = get_node("/root/DatabaseOperations")
+
+
+func _load_profession_screen():
```

Comment:
make it _load_screen - it is already in profession script. And that would allow us to call it without specifying object.

Reply:
You fixed that :)